content/src/content/jcr_root/apps/core/wcm/components/title/v3/title/_cq_design_dialog/.content.xml

```diff
@@ -41,70 +42,60 @@
                                 jcr:primaryType="nt:unstructured"
                                 sling:resourceType="granite/ui/components/coral/foundation/container">
                                 <items jcr:primaryType="nt:unstructured">
-                                    <inputgroup
+                                    <allowedHeadingElements
                                         jcr:primaryType="nt:unstructured"
-                                        sling:resourceType="granite/ui/components/coral/foundation/text"
-                                        text="Allowed Types / Sizes for Authors"/>
-                                    <field
-                                        cq:showOnCreate="{Boolean}true"
-                                        granite:class="core-title-sizes-allowed"
-                                        jcr:primaryType="nt:unstructured"
-                                        sling:resourceType="granite/ui/components/coral/foundation/form/fieldset"
-                                        fieldDescription="The HTML element to use for the download title. If no type is configured, the default (H3) will be used.">
+                                        sling:resourceType="granite/ui/components/coral/foundation/form/select"
+                                        fieldDescription="The heading HTML elements that are allowed to be selected by an author for the title type."
+                                        fieldLabel="Allowed Heading Elements"
+                                        multiple="{Boolean}true"
+                                        name="./allowedTypes"
+                                        ordered="{Boolean}true"
+                                        required="{Boolean}true">
                                         <items jcr:primaryType="nt:unstructured">
                                             <h1
                                                 jcr:primaryType="nt:unstructured"
-                                                sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
-                                                checked="true"
-                                                name="./allowedTypes"
-                                                text="H1"
+                                                selected="{Boolean}true"
+                                                text="h1"
```

Comment:
Shouldn't it be: H1, H2, ... with capital H?

Reply:
I don't think so, we have to change the UX of the title policy dialog to be consistent with the UX of the accordion policy dialog (which utilises small letters).
<img width="1718" alt="Screenshot 2022-04-28 at 14 10 09" src="https://user-images.githubusercontent.com/79856745/165739676-9df6df14-200b-4146-b685-169b81a16c88.png">
.

Reply:
@jckautzmann Didn't know about the TOC discussion. Updated here 13d96bdf1c555a16ff9a06c3a5058ebc7ad8610b the edit and design dialog for Accordion, Title and Teaser, to be consistent.

Reply:
thx @comanV !